### PR TITLE
Retrieve messages guide

### PR DESCRIPTION
**Pull request by daphsta**
This is an additional guide to add on extra functionality to the current app.
Appreciate a review on this :) 
This PR references [this issue](https://github.com/Elixir-Girls/elixir-girls.github.io/issues/16)

### CODE REVIEW

File: source/guide1.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Elixir Girls Guides Slackir App retrieving data
+---
+
+<%= partial "partials/guides-nav" %>
```
Comment (brentonannan) [inline source/guide1.html]: If you rebase onto master, and put this into `source/guides/`, you don't need wrapping stuff, it's in a layout.

File: source/guide1.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+          <% end %>
+        </div>
+        <!-- /#page-wrapper -->
+
+    </div>
+    <!-- /#wrapper -->
+
+
+<!-- jQuery -->
```
Comment (brentonannan) [inline source/guide1.html]: Also in a partial.

File: source/guide1.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                if authorized?(payload) do
+                  send self(), :after_join #<-- Add this line
+
+            <% end %>
+
+          <h4>What does send do?</h4>
+          <p><em>send</em> manually sends a message <em>:after_join</em> with it's PID generated by the function <em>self()</em>. We want to be able to run the after join task asynchronously. Lets look at how we should receive the :after_join message</p>
+          <h1>Handling a message in a callback</h1>
+          <p></p>
+          <% code("elixir") do %>
```
Comment (brentonannan) [inline source/guide1.html]: I *so* wish I'd known about this helper sooner.

**Review by brentonannan — CHANGES_REQUESTED** [review]
Produced `html` files are named after these files, so well-named files would be useful for SEO etc.

Also, these should be `.html.erb` files.

... I just looked further - seems you have a mix of files in here, not sure what's what. Also, the `persistence` guide seems to be re-added under `source`.

**Comment by daphsta** [comment]
@brentonannan opps, my bad for committing an incomplete file. That's all fixed now

**Comment by daphsta** [comment]
@catherine-jones @brentonannan is this good to go?

**Comment by brentonannan** [comment]
Hey, sorry @daphsta, I had a hella crazy day today and didn't get a chance to review :disappointed:.

**Comment by daphsta** [comment]
No worries @brentonannan, totally understand that :)

**Pull request merged**